sycl/plugins/unified_runtime/CMakeLists.txt

```diff
@@ -56,14 +56,14 @@ endif()
 if(SYCL_PI_UR_USE_FETCH_CONTENT)
   include(FetchContent)
 
-  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
-  # commit d216eb44d5c9fe3433eecdd09b10e3e79ac25bd7
-  # Merge: 40517d2b fc1f3066
-  # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
-  # Date:   Wed Jan 31 10:38:07 2024 +0000
-  #     Merge pull request #1226 from hdelan/get-native-mem-on-device2
-  #     [UR] Add extra param to urMemGetNativeHandle
-  set(UNIFIED_RUNTIME_TAG d216eb44d5c9fe3433eecdd09b10e3e79ac25bd7)
+  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime")
+  # commit c6b0a240c40dd385631d03315f3b9d020bf31e3a
+  # Merge: 5d5c8106 0631330a
+  # Author: Piotr Balcer <piotr.balcer@intel.com>
+  # Date:   Thu Feb 1 10:47:43 2024 +0100
+  #     Merge pull request #1210 from AllanZyne/sanitizer-pr-cpu-local
+  #     [DeviceSanitizer] Support detecting out-of-bounds error on CPU Device & Static Local Memory
+  set(UNIFIED_RUNTIME_TAG c6b0a240c40dd385631d03315f3b9d020bf31e3a)
```

Comment:
The sycl branch now points at 9363574db721d2388c7d76a10edb128764872352 which is 2 commits ahead of c6b0a240c40dd385631d03315f3b9d020bf31e3a. This file is also currently conflicting, so when resolving that conflict please ensure that 9363574db721d2388c7d76a10edb128764872352 is being used.

```log
commit 9363574db721d2388c7d76a10edb128764872352
Merge: 553a6b82 5e513738
Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
Date:   Thu Feb 1 11:50:16 2024 +0000

    Merge pull request #1302 from kbenzie/benie/cl-binary-type-intermediate

    [CL] Handle INTERMEDIATE binary type

commit 553a6b8213098f51858a814b9f45e989af82da93
Merge: c6b0a240 163e8235
Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
Date:   Thu Feb 1 10:33:59 2024 +0000

    Merge pull request #1301 from kbenzie/benie/update-actions-checkout

    [GHA] Bump actions/checkout to v4.1.1

commit c6b0a240c40dd385631d03315f3b9d020bf31e3a
Merge: 5d5c8106 0631330a
Author: Piotr Balcer <piotr.balcer@intel.com>
Date:   Thu Feb 1 10:47:43 2024 +0100

    Merge pull request #1210 from AllanZyne/sanitizer-pr-cpu-local

    [DeviceSanitizer] Support detecting out-of-bounds error on CPU Device & Static Local Memory
```

Reply:
Okay, thanks!